tests/data/data.ts

```diff
@@ -448,11 +448,13 @@ export class TestDataProvider {
     return retVal;
   }
 
-  static getAggregationPipeline(
-    matchConditionField: string,
-    matchConditionValue: string,
-    projection?: { [k: string]: number }
-  ) {
+  static getAggregationPipeline(matchConditions: { [k: string]: string }, projection?: { [k: string]: number }) {
+    // Add prefix 'repo' to each field in matchConditions
+    const formattedMatchConditions = Object.entries(matchConditions).reduce((acc, [key, val]) => {
+      acc[`repo.${key}`] = val;
+      return acc;
+    }, {});
```

Comment:
Is this always expected to be the same as `DocsetsRepository`'s function? If so, would it be worth making `DocsetsRepository`'s `getAggregationPipeline` a static function or just exporting it separately, and using it directly in the tests to avoid needing duplicate code?